src/flows/pipes/Notification/endpoints/PagerDuty/index.ts

```diff
@@ -29,7 +29,7 @@ export default register => {
 		messageFn: messageFn,
     crit: trigger,
 	)
-	|> monitor["notify"](data: notification, endpoint: pagerduty["endpoint"]()(mapFn: (r) => ({
+	|> monitor["notify"](data: notification, endpoint: pagerduty.endpoint()(mapFn: (r) => ({ r with
```

Comment:
not sure how I feel about the `r with` object extension.
It matches the docs...but it does potentially push client data over the wire.

Reply:
confirmed. This is the intent. The alert templating system is then responsible for deciding.